ui/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/ProcessRepository.scala

```diff
@@ -98,12 +98,18 @@ class DBProcessRepository(val dbConfig: DbConfig, val modelVersion: ProcessingTy
       case CustomProcess(mainClass) => (None, Some(mainClass))
     }
 
+    def normalizeJsonString(jsonString: String) = jsonString.filterNot(_.isWhitespace)
```

Comment:
let's compare parsed json instead (i.e. Json class). Otherwise we will still have possible issues - like changing order of fields in json object etc.  I think we can actually pass Json instead of raw string in some places (although I would do large refactor ATM :))